client/src/components/ButtonRowPicker/ButtonRowPicker.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ModalSelector from 'react-native-modal-selector';
+
+
+import { View } from 'react-native';
+import { ButtonRow } from '../Button';
+
+class ButtonRowPicker extends Component {
+  onConfirm = (option) => {
+    this.props.selected = option;
+    this.props.onSelect(option);
+  };
+
+  render() {
+    const data = [
+      { key: 0, label: 'Available' },
+      { key: 1, label: 'Unavailable' },
+      { key: 2, label: 'Unavailable - unless urgent' },
+    ];
+
+    return (
+      <View>
+        <ModalSelector
+          data={data}
+          initValue="Select something yummy!"
+          onChange={(option) => { this.onConfirm(option.label); }}
```

Comment:
should try to avoid creating a closure inside render - just use:
```
onChange={this.onConfirm}
```
then modify `onConfirm` to get at `label`.

Reply:
ty